lisp/ess-dde.el

```diff
@@ -96,7 +96,7 @@ PROC, VISIBLY and MESSAGE are ignored."
 If prefix arg is given, forces a query of the ESS process for the help
```

Comment:
I think no-one is using dde these days. I really doubt it works as originally intended. 

Maybe it's time to move this part to `./old/`?

Reply:
That would make sense to me. I think another candidate is the mouseme stuff. Or do people use that?